hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1054,33 +1054,34 @@
       tests: ["chip_sw_rstmgr_sw_req"]
     }
     {
-      name: chip_sw_rstrmgr_alert_info
+      name: chip_sw_rstmgr_alert_info
       desc: '''Verify the expected values from the `alert_info` CSR on reset.
 
             Various alerts can be created, for example, timeouts, and integrity errors, and at
             least part of the `alert_info` CSR can be predicted. To cause some of these to
             cause a reset, mask the relevant processor interrupts. Trigger these resets and
             verify the `alert_info` register contents when reset is handled.
-            Refer to `chip_*sys_rstrmgr_reset_info`.
+            Refer to `chip_*sys_rstmgr_reset_info`.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_sw_rstrmgr_sw_rst
+      name: chip_sw_rstmgr_sw_rst
       desc: '''Verify `sw_rst_ctrl_n` CSR resets individual peripherals.
 
-            Check the peripheral is reset via timeout accessing their CSRs while the corresponding
-            reset is active. The per peripheral reset controls in increasing order by
-            `sw_rst_ctrl_n` bit, are as follows: `spi_device`, `spi_host0`, `spi_host0_core`,
-            `spi_host1`, `spi_host1_core`, `usb`, `usbif`, `i2c0`, `i2c1`, `i2c2`.
+            Pick a rw type resister in each peripheral and program arbitrary value.
+            Then read back the rw resister after reset.
+            Check the value returns to reset value.
+            The per peripheral reset controls in increasing order by
+            `sw_rst_ctrl_n` bit, are as follows: `spi_device`, `spi_host0`, `spi_host1`,
+            `usb`, `usbif`, `i2c0`, `i2c1`, `i2c2`.
```

Comment:
Nit: suggested edits; recommend making this a bulleted list:
```suggestion
            - Pick a  register with RW access policy, in each peripheral and write an 
              arbitrary safe value (that does not cause any adverse side-effects).
            - Pulse the reset to the peripheral via software. 
            - Read the register after reset and verify it returns the reset value.
            - Repeat these steps for each of these software resettable peripherals:
              `spi_device`, `spi_host0`, `spi_host1`, `usb`, `usbif`, `i2c0`, `i2c1`, `i2c2`.
```

Reply:
updated